tests/lib/rules/newline-before-return.js

```diff
@@ -112,77 +112,90 @@ ruleTester.run("newline-before-return", rule, {
     invalid: [
         {
             code: "function a() {\nvar b;\nreturn;\n}",
-            errors: ["Expected newline before return statement."]
+            errors: ["Expected newline before return statement."],
+            output: "function a() {\nvar b;\n\nreturn;\n}"
         },
         {
             code: "function a() {\nif (b) return b;\nelse if (c) return c;\nelse {\ne();\nreturn d;\n}\n}",
-            errors: ["Expected newline before return statement."]
+            errors: ["Expected newline before return statement."],
+            output: "function a() {\nif (b) return b;\nelse if (c) return c;\nelse {\ne();\n\nreturn d;\n}\n}"
         },
         {
             code: "function a() {\n while (b) {\nc();\nreturn;\n}\n}",
-            errors: ["Expected newline before return statement."]
+            errors: ["Expected newline before return statement."],
+            output: "function a() {\n while (b) {\nc();\n\nreturn;\n}\n}"
         },
         {
             code: "function a() {\ndo {\nc();\nreturn;\n} while (b);\n}",
-            errors: ["Expected newline before return statement."]
+            errors: ["Expected newline before return statement."],
+            output: "function a() {\ndo {\nc();\n\nreturn;\n} while (b);\n}"
         },
         {
             code: "function a() {\nfor (var b; b < c; b++) {\nc();\nreturn;\n}\n}",
-            errors: ["Expected newline before return statement."]
+            errors: ["Expected newline before return statement."],
+            output: "function a() {\nfor (var b; b < c; b++) {\nc();\n\nreturn;\n}\n}"
         },
         {
             code: "function a() {\nfor (b in c) {\nd();\nreturn;\n}\n}",
-            errors: ["Expected newline before return statement."]
+            errors: ["Expected newline before return statement."],
+            output: "function a() {\nfor (b in c) {\nd();\n\nreturn;\n}\n}"
         },
         {
             code: "function a() {\nfor (b of c) {\nd();\nreturn;\n}\n}",
             parserOptions: { ecmaVersion: 6 },
-            errors: ["Expected newline before return statement."]
+            errors: ["Expected newline before return statement."],
+            output: "function a() {\nfor (b of c) {\nd();\n\nreturn;\n}\n}"
         },
         {
             code: "function a() {\nif (b) {\nc();\n}\n//comment\nreturn b;\n}",
-            errors: ["Expected newline before return statement."]
-        },
-        {
-            code: "function a() {\nif (b) {\nc();\n}\n//comment\nreturn b;\n}",
-            errors: ["Expected newline before return statement."]
+            errors: ["Expected newline before return statement."],
+            output: "function a() {\nif (b) {\nc();\n}\n//comment\nreturn b;\n}"
```

Comment:
In some tests (like this one), I've added an `output` even though nothing changed, to guarantee that a fix is not applied in this case. Hope that's the correct approach!